sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Category.cs

```diff
@@ -1,12 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     /// <summary>
     /// A class contains constant strings that represent SignalR trigger category.
     /// </summary>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "AZC0012:Avoid single word type names", Justification = "Breaking change")]
+    [Obsolete("Please use SignalRTriggerCategories instead.")]
```

Comment:
We don't obsolete stuff due to the potential for it to cause breaking changes. Instead we use the EditorBrowsable attribute and set to Never.